llama_index/llms/replicate.py

```diff
@@ -27,11 +27,13 @@ def __init__(
         context_window: int = DEFAULT_CONTEXT_WINDOW,
```

Comment:
I think we should remove the `max_tokens` config option now? Since it's no longer passed anywhere.

Reply:
That's fair!